ui/components/ReconciledObjectsTable.tsx

```diff
@@ -25,6 +76,14 @@ function ReconciledObjectsTable({
     false
   );
 
+  //calculate canary status
+  useEffect(() => {
```

Comment:
This is effectively setting parent status from a child component, which we want to try and avoid. Instead, we should be lifting this logic up to where it is needed.

I realize this is a result of how the component tree is laid out.

Reply:
Yeah the reconciled objects req was recently moved down into the individual tabs. Do we care if I move it back out or nah?